from PyQt6 import QtCore, QtGui, QtWidgets


class Ui_bank_account(object):
    def setupUi(self, bank_account):
        bank_account.setObjectName("bank_account")
        bank_account.setEnabled(True)
        bank_account.resize(679, 473)
        bank_account.setMaximumSize(QtCore.QSize(1000, 1000))
        bank_account.setWindowTitle("Bank Account")
        self.centralwidget = QtWidgets.QWidget(parent=bank_account)
        self.centralwidget.setMaximumSize(QtCore.QSize(1000, 1000))
        self.centralwidget.setObjectName("centralwidget")
        self.bank_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.bank_label.setGeometry(QtCore.QRect(300, 20, 55, 16))
        self.bank_label.setObjectName("bank_label")
        self.first_name_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.first_name_label.setGeometry(QtCore.QRect(180, 50, 81, 20))
        self.first_name_label.setObjectName("first_name_label")
        self.last_name_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.last_name_label.setGeometry(QtCore.QRect(184, 80, 71, 20))
        self.last_name_label.setObjectName("last_name_label")
        self.label_4 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_4.setGeometry(QtCore.QRect(184, 110, 71, 20))
        self.label_4.setObjectName("label_4")
        self.login_button = QtWidgets.QPushButton(parent=self.centralwidget)
        self.login_button.setGeometry(QtCore.QRect(430, 80, 93, 28))
        self.login_button.setObjectName("login_button")
        self.label_5 = QtWidgets.QLabel(parent=self.centralwidget)
        self.label_5.setGeometry(QtCore.QRect(250, 170, 161, 16))
        self.label_5.setObjectName("label_5")
        self.withdraw_button = QtWidgets.QCheckBox(parent=self.centralwidget)
        self.withdraw_button.setGeometry(QtCore.QRect(140, 270, 95, 20))
        self.withdraw_button.setObjectName("withdraw_button")
        self.saving_acc_button = QtWidgets.QCheckBox(parent=self.centralwidget)
        self.saving_acc_button.setGeometry(QtCore.QRect(260, 200, 151, 20))
        self.saving_acc_button.setObjectName("saving_acc_button")
        self.checking_acc_button = QtWidgets.QCheckBox(parent=self.centralwidget)
        self.checking_acc_button.setGeometry(QtCore.QRect(260, 230, 151, 20))
        self.checking_acc_button.setObjectName("checking_acc_button")
        self.name_input = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.name_input.setGeometry(QtCore.QRect(270, 50, 113, 22))
        self.name_input.setObjectName("name_input")
        self.last_label = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.last_label.setGeometry(QtCore.QRect(270, 80, 113, 22))
        self.last_label.setObjectName("last_label")
        self.pin_input = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.pin_input.setGeometry(QtCore.QRect(270, 110, 113, 22))
        self.pin_input.setObjectName("pin_input")
        self.deposit_button = QtWidgets.QCheckBox(parent=self.centralwidget)
        self.deposit_button.setGeometry(QtCore.QRect(240, 270, 95, 20))
        self.deposit_button.setObjectName("deposit_button")
        self.amount_input = QtWidgets.QLineEdit(parent=self.centralwidget)
        self.amount_input.setGeometry(QtCore.QRect(420, 270, 113, 22))
        self.amount_input.setObjectName("amount_input")
        self.amount_label = QtWidgets.QLabel(parent=self.centralwidget)
        self.amount_label.setGeometry(QtCore.QRect(344, 270, 71, 20))
        self.amount_label.setObjectName("amount_label")
        self.submit_button = QtWidgets.QPushButton(parent=self.centralwidget)
        self.submit_button.setGeometry(QtCore.QRect(430, 300, 93, 28))
        self.submit_button.setObjectName("submit_button")
        self.exit_button = QtWidgets.QPushButton(parent=self.centralwidget)
        self.exit_button.setGeometry(QtCore.QRect(330, 300, 93, 28))
        self.exit_button.setObjectName("pushButton")
        bank_account.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(parent=bank_account)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 679, 26))
        self.menubar.setObjectName("menubar")
        bank_account.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(parent=bank_account)
        self.statusbar.setObjectName("statusbar")
        bank_account.setStatusBar(self.statusbar)

        self.retranslateUi(bank_account)
        QtCore.QMetaObject.connectSlotsByName(bank_account)

    def retranslateUi(self, bank_account):
        _translate = QtCore.QCoreApplication.translate
        self.bank_label.setText(_translate("bank_account", "BANK"))
        self.first_name_label.setText(_translate("bank_account", "First Name"))
        self.last_name_label.setText(_translate("bank_account", "Last Name"))
        self.label_4.setText(_translate("bank_account", "PIN Number"))
        self.login_button.setText(_translate("bank_account", "Log In"))
        self.label_5.setText(_translate("bank_account", "What would you like to do?"))
        self.withdraw_button.setText(_translate("bank_account", "Withdraw"))
        self.saving_acc_button.setText(_translate("bank_account", "SAVINGS ACCOUNT"))
        self.checking_acc_button.setText(_translate("bank_account", "CHECKING ACCOUNT"))
        self.deposit_button.setText(_translate("bank_account", "Deposit"))
        self.amount_label.setText(_translate("bank_account", " AMOUNT $"))
        self.submit_button.setText(_translate("bank_account", "submit"))
        self.exit_button.setText(_translate("bank_account", "Exit"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    bank_account = QtWidgets.QMainWindow()
    ui = Ui_bank_account()
    ui.setupUi(bank_account)
    bank_account.show()
    sys.exit(app.exec())
